component/src/main/java/org/wso2/extension/siddhi/execution/reorder/AlphaKSlackExtension.java

```diff
@@ -60,114 +60,118 @@
                         name = "beta1",
-                        description = "An upper limit value assigned for the unsuccessfully handled late arrivals",
+                        description = "An upper limit value assigned for the unsuccessfully handled late arrivals.",
                         type = {DataType.DOUBLE}
                 ),
                 @ReturnAttribute(
                         name = "beta2",
                         description = "The number of events that should be considered in the calculation of an alpha " +
-                                "value",
+                                "value.",
                         type = {DataType.LONG}
                 ),
                 @ReturnAttribute(
                         name = "beta3",
-                        description = "Fixed time-out value (in milliseconds) assigned for flushing all the events " +
-                                "buffered inside the extension.",
+                        description = "The fixed time-out value in milliseconds assigned for flushing all the " +
+                                "events buffered inside the extension.",
                         type = {DataType.LONG}
                 ),
                 @ReturnAttribute(
                         name = "beta4",
-                        description = "Maximum threshold value assigned for K parameter.",
+                        description = "The maximum threshold value assigned for 'K' parameter.",
                         type = {DataType.LONG}
                 ),
                 @ReturnAttribute(
                         name = "beta5",
-                        description = "Flag set to indicate whether out-of-order events which arrive after buffer " +
-                                "eviction to be discarded or not",
+                        description = "The flag set to indicate whether the out-of-order events which arrive " +
+                                "after buffer eviction are to be discarded or not.",
                         type = {DataType.BOOL}
                 ),
                 @ReturnAttribute(
                         name = "beta6",
-                        description = "Error threshold value set for Alpha K-Slack algorithm",
+                        description = "The error threshold value set for Alpha K-Slack algorithm.",
                         type = {DataType.DOUBLE}
                 ),
                 @ReturnAttribute(
                         name = "beta7",
-                        description = "Confidence level set for the Alpha K-Slack algorithm",
+                        description = "The confidence level set for the Alpha K-Slack algorithm.",
                         type = {DataType.DOUBLE}
                 )
         },
         examples = @Example(
-                syntax = "define stream inputStream (eventtt long,data double);\n" +
+                syntax = "define stream InputStream (eventtt long,data double);\n" +
```

Comment:
We cannot change attribute name only in the query, stream definition also needs to be changed.
```suggestion
                syntax = "define stream InputStream (event long,data double);\n" +
```

Reply:
Fixed in 6fbd6f6